AdobeIms/Test/Unit/Model/ConfigTest.php

```diff
@@ -36,28 +35,121 @@ class ConfigTest extends TestCase
     private $scopeConfigMock;
 
     /**
-     * Prepare test objects.
+     * @var UrlInterface|MockObject
+     */
+    private $urlMock;
+
+    /**
+     * Set up test mock objects
      */
     protected function setUp(): void
     {
-        $this->objectManager = new ObjectManager($this);
         $this->scopeConfigMock = $this->createMock(ScopeConfigInterface::class);
-        $this->config = $this->objectManager->getObject(
-            Config::class,
-            [
-                'scopeConfig' => $this->scopeConfigMock
-            ]
-        );
+        $this->urlMock = $this->createMock(UrlInterface::class);
+
+        $this->config = new Config($this->scopeConfigMock, $this->urlMock);
```

Comment:
Why you removed objectManager and ccreate class directly ?

Reply:
The only point of having an object manager helper is to mock some dependencies for you. Especially when you are a complex object with a lot of dependencies. If you mock everything by yourself there is no reason to use it.

Also from technical guidelines:
> 11.2.3. ObjectManagerHelper MAY BE used to automatically mock all dependencies of the object under test.
Which means that it's not required.

Reply:
@Nazar65 @hws47a @drpayyne ObjectManagerHelper is always preferable for unit tests. It prevents the test from failing if additional dependency is introduced to the class under test and also makes further work with class and test easier.

Reply:
agree